requirements/test.txt

```diff
@@ -25,3 +25,4 @@ pytest==3.0.3             # via pytest-catchlog, pytest-cov, pytest-django
 python-dateutil==2.5.3    # via fake-factory, freezegun
 six==1.10.0               # via django-extensions, fake-factory, freezegun, mock, python-dateutil
 unicodecsv==0.14.1
+mock==2.0.0
```

Comment:
@asadiqbal08 sorry for not noticing it earlier, but there's already `mock==2.0.0` in this file on line 17 - I believe you should remove this line.